invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/modal_manager/ModalManager.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import { ModalContext } from "./index";
+import PropTypes from "prop-types";
+
+export class ModalManager extends Component {
```

Comment:
this manager layer could be potentially avoided if modals manage their own state. If you place the modal itself as a wrapper for `<ResultList>`, you need only to pass userID to it via `Remove user` button callback. The context is not needed in this case IMO and it overcomplicates the code. Please reach out if the description  of the proposed solution is not clear

Reply:
But how will the callback be called inside the resultItem and then transfered to the modal outside `ResultsList` :thinking: